docs/solidity-by-example.rst

```diff
@@ -137,6 +137,14 @@ of votes.
             sender.voted = true;
             sender.delegate = to;
             Voter storage delegate_ = voters[to];
+
+            //Check if the delegate will vote for the 'same proposal' 
+            //as requested by the sender - otherwise, the number of votes
+            //will be added to the delegate's choice of proposal, 
+            //instead of the sender's choice of proposal.
+            
+            require(delegate_.vote == proposalChoice, "Delegate is voting for a 'different proposal'");
```

Comment:
Let's assume that the **delegate_.vote** referes to **proposal A**, and 
the sender's **proposalChoice** referes to **proposal B**. 

Without the following check,

`require(delegate_.vote == proposalChoice, "Delegate is voting for a 'different proposal'");`

the **delegated votes would wrongly be added to that of 'Proposal A' instead of 'Proposal B'**.